runscope/data_source_runscope_integration_test.go

```diff
@@ -4,21 +4,22 @@ import (
 	"fmt"
 	"testing"
 
```

Comment:
not a big deal here, `goimports` sometimes doesn't quite get the whitespacing right as it errs on the side of including your import groups, but I'd group `os` with the other stdlib packages normally, but won't hold up process on this.